bin/phraseapp_updater

```diff
@@ -41,8 +42,10 @@ class PhraseAppUpdaterCLI < Thor
       puts "Bad PhraseApp API key."
     rescue PhraseAppUpdater::PhraseAppAPI::BadProjectIDError => e
       puts "Bad PhraseApp project ID: #{phraseapp_project_id}"
+    rescue PhraseAppUpdater::LocaleFile::BadFileTypeError => e
```

Comment:
Yeah, I was thinking of doing something more complicated, like registering handlers for each file type. Then I could iterate through all of them to show the accepted file types. An enum probably works for now though.